src/components/vm/nics/nicBody.jsx

```diff
@@ -174,7 +174,7 @@ export const NetworkTypeAndSourceRow = ({ idPrefix, onValueChanged, dialogValues
                                             </p>
                                         </TextContent>))}
                                     </Flex>}>
-                               <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                               <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230630-122011-699fe1a0-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)